requirements.txt

```diff
@@ -1,2 +1,2 @@
-sphinx==1.4.9
-git+https://github.com/StackStorm/sphinx-autobuild.git@develop
+sphinx
```

Comment:
Personally I would still a bit better if we at least pin it to the latest major version - `sphinx>=1.5.3,<1.6`.

This way we can avoid surprises if a new version pushes a backward incompatible or breaking change (we had this happened many times in st2 when we didn't pin dependencies to specific version and we wasted a lot of times debugging such issues).